stats/Exemplars.md

```diff
@@ -0,0 +1,48 @@
+Exemplars
+===
+
+Exemplars are example data points for aggregated data. In particular, for distribution-type
+metrics, exemplars are points associated with each bucket in the distribution giving an
+example of what was aggregated into the bucket.
+
+Exemplars consist of a value that was recorded, an exact timestamp of when that value
+was recorded, and a set of attachments.
+
+Attachments
+---
+
+Exemplars my have zero or more attachments. Attachments are key-value pairs that
+describe the context in which the exemplar was recored.
+
+Well-known attachment keys
+---
+
+We define the following types of attachments:
+
+1. `TraceID`: lower-case base16 encoded trace ID
+2. `SpanID`:  lower-case base16 encoded span ID
+3. `TraceOptions`: lower-case base16 encoded trace options
+4. `Tag:KEY`: tags that were active at the time of recording, regardless of whether
```

Comment:
what about `tracestate`?

Reply:
Not sure how to handle it. For now I recommend we leave it out.

Reply:
To clarify, I don't really have good knowledge on what tracestate is - open to suggestions on alternative ways to handle it here.

Reply:
I'd say there should be a callback to allow people to extract more, especially from `tracestate`

Reply:
I will add: 
```
Implementations should provide an extensible mechanism to extract custom attachments
from the context.
```